src/store/channels-list/saga.ts

```diff
@@ -91,6 +92,9 @@ export function* fetchConversations() {
 
   const channel = yield call(getConversationsBus);
   yield put(channel, { type: ConversationEvents.ConversationsLoaded });
+
+  // Validate the active conversation after conversations are loaded
+  yield call(performValidateActiveConversation);
```

Comment:
The follow up PR for displaying the error dialog will handle performing this validation check when the url/room id changes.